### PR TITLE
Add Short Tooltip Description for Some AE2 Items

### PR DESCRIPTION
**Pull request by v3ect0rgames**

### CODE REVIEW

File: overrides/scripts/AE2.zs [hunk 1]
```diff
@@ -6,12 +6,22 @@ import mods.gregtech.recipe.RecipeMap;
...
+<appliedenergistics2:material:58>.addTooltip(format.yellow("Used in interfaces. Allows you to put more patterns in an interface then normally allowed up to 4x the normal amount of patterns. Check the Quest Book for more information.")); //pattern expansion card
+<appliedenergistics2:material:53>.addTooltip(format.yellow("Allows machines to craft items as they are used up automatically.")); //crafting card
+<appliedenergistics2:material:30>.addTooltip(format.yellow("Speeds up various ae2 processes such as exporting things into a machine, speeding up molecular accelerators.")); //acceleration card
+<appliedenergistics2:material:60>.addTooltip(format.yellow("When put in a wireless terminal, it will pick items off the floor and put them into your ae2 system based on your configuration.")); //magnet card
+<appliedenergistics2:material:31>.addTooltip(format.yellow("Inverts ae2 configurations. Example: If you have a storage bus set to only store Omnium Ingots, and you have this card, it will store every item but Omnium Ingots")); //inverter card
```
Comment (tracer4b) [inline overrides/scripts/AE2.zs:14]: Capitalize AE2 and remove the examples. Examples should be put in the qb (doesn't need to be added in this PR)

**Review by tracer4b — CHANGES_REQUESTED** [review]

File: overrides/scripts/AE2.zs (hunk 1 above)
Comment (v3ect0rgames) [reply overrides/scripts/AE2.zs]: Fixed it

Comment (v3ect0rgames) [reply overrides/scripts/AE2.zs]: can you merge it now

**Comment by v3ect0rgames** [comment]
@IntegerLimit Once this is merged it should complete an item on the 1.7 todo

**Comment by IntegerLimit** [comment]
I'm very hesitant on adding further tooltips to the current CT scripts right now. This is because we need to migrate all tooltips to groovy, as well as allow localisation of current tooltips.

Furthermore, for some of these tooltips, such as the sticky card, a simple tooltip seems not enough, and a JEI description page may fit better.

However, the tooltips on this PR do seem like a good starting point. For that reason, I will merge this PR, but the proper tooltip porting will have to be done later. (I will do it in my upcoming Labs Update + GTCEu Update PR)

Thank you for this PR!

**Review by IntegerLimit — APPROVED** [review]
As base tooltips, this is appropriate for the project.

**Pull request merged**